cvat-core/src/server-proxy.js

```diff
@@ -595,6 +595,11 @@
                 }
 
                 const taskData = new FormData();
+
+                const clientFile = taskDataSpec.client_files[0];
+                const clientFiles = taskDataSpec.client_files;
```

Comment:
Just newer syntax to get the first element of an array is:

const clientFiles = taskDataSpec.client_files;
const [clientFile] = clientFiles;

Reply:
Done.